api/src/app/modules/project/project.service.ts

```diff
@@ -784,6 +784,105 @@ export class ProjectService extends DataService<Project, Repository<Project>, Pr
     return true;
   }
 
+  /**
+   * Counts the number of projects where commentingOpenDate falls between startDate and endDate. 
+   * Used by analytics dashboard module.
+   * 
+   * @param startDate - The start of the date range (inclusive, YYYY-MM-DD)
+   * @param endDate - The end of the date range (inclusive, YYYY-MM-DD)
+   * @returns Promise resolving to the count of projects matching the criteria
+   */
+  async getProjectCountByDate(startDate: string, endDate: string): Promise<number> {
+    return await this.repository
+      .createQueryBuilder()
```

Comment:
Could add a convenience method to create a query builder already filtering by start & end date and excluding initial FOMs, reuse it across these different service methods.